test/ipynb/mpl/graph/test_graph_matplotlib_drawer.py

```diff
@@ -370,6 +370,31 @@ def test_plot_coupling_map(self):
             filename="coupling_map.png",
         )
 
+    def test_plot_bloch_multivector_figsize_improvements(self):
+        """test bloch sphere figsize, font_size, title_font_size and title_pad
+        See https://github.com/Qiskit/qiskit-terra/issues/7263
+        and https://github.com/Qiskit/qiskit-terra/pull/7264.
+        """
+        circuit = QuantumCircuit(3)
+        circ.h(1)
+        circ.sxdg(2)
```

Comment:
This is `NameError`, I think
```suggestion
        circuit.h(1)
        circuit.sxdg(2)
```